Server/src/Server.Domain/Entities/Review.cs

```diff
@@ -0,0 +1,26 @@
+// ReSharper disable CollectionNeverUpdated.Global
+#pragma warning disable CS8618
+
+namespace Server.Domain.Entities;
+
+public class Review
+{
+    public Guid Id { get; set; }
+    public string? Title { get; set; }
+    public string? Description { get; set; }
+    public double Rating { get; set; }
+    public Book Book { get; set; }
+    //public User Author { get; set; }
```

Comment:
A no to by się przydało, w sensie żeby było `IdentityId` i `Identity`. Jak sobie spojrzysz do #89 to tam w ControllerBase.cs dodałem taką metodę do pobierania Id zalogowanego użytkownika

Reply:
Ok, to zrobie nowe issue na pelna obsluge userow w tym